articles/quickstart/native/xamarin-ios/01-login.md

```diff
@@ -0,0 +1,171 @@
+
+This is an example of the XML representation of your `info.plist` file after you have added the URL Type:
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+    <dict>
+        <key>CFBundleTypeRole</key>
+        <string>None</string>
+        <key>CFBundleURLName</key>
+        <string>Auth0</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+            <string>YOUR_BUNDLE_IDENTIFIER</string>
+        </array>
+    </dict>
+</array>
+```
+
+Next, go to your [Client's Dashboard](${manage_url}/#/applications/${account.clientId}/settings) and make sure that *Allowed Callback URLs* contains the following:
+
+```text
+YOUR_BUNDLE_IDENTIFIER://${account.namespace}/ios/YOUR_BUNDLE_IDENTIFIER/callback
+```
+
+Replace `YOUR_BUNDLE_IDENTIFIER` in the code sample above with the actual Bundle Identifier for your application, e.g. `com.mycompany.myapplication`.
+
+Finally, you will need to handle the callback to your application in your `AppDelegate` by calling the `Send` method of the `ActivityMediator` singleton, passing along the url sent in. This will allow the Auth0 OIDC Client library to complete the authentication process:
+
+```csharp
+using Auth0.OidcClient;
+
+[Register("AppDelegate")]
+public class AppDelegate : UIApplicationDelegate
+{
+	public override bool OpenUrl(UIApplication application, NSUrl url, string sourceApplication, NSObject annotation)
+	{
+		ActivityMediator.Instance.Send(url.AbsoluteString);
+
+		return true;
+	}
+}
+```
+
+## Integration
+
+To integrate Auth0 login into your application, simply instantiate an instance of the `Auth0Client` class, configuring the Auth0 Domain and Client ID, and also passing the instance uf your `UIViewController` as the `Controller` property: 
+
+${snippet(meta.snippets.setup)}
+
+You can then call the `LoginAsync` method to log the user in:
+
+${snippet(meta.snippets.use)}
+
+<div class="phone-mockup"><img src="/media/articles/native-platforms/xamarin-ios/lock-widget-screenshot.png" alt="Lock UI"></div>
+
+## Accessing the User's Information
+
+The returned login result will indicate whether authentication was successful, and if so contain the tokens and claims of the user.
+
+### Authentication Error
+
+You can check the `IsError` property of the result to see whether the login has failed. The `ErrorMessage` will contain more information regarding the error which occurred.
+
+```csharp
+var loginResult = await client.LoginAsync();
+
+if (loginResult.IsError)
+{
+    Debug.WriteLine($"An error occurred during login: {loginResult.Error}")
+}
+```
+
+### Accessing the tokens
+
+On successful login, the login result will contain the `id_token` and `access_token` in the `IdentityToken` and `AccessToken` properties respectively.
+
+```csharp
+var loginResult = await client.LoginAsync();
+
+if (!loginResult.IsError)
+{
+    Debug.WriteLine($"id_token: {loginResult.IdentityToken}");
```

Comment:
Mixing of case = C# naming conventions
Naming of `IdentityToken` is because that is what the underlying library I am using decided to call it. I am returning their result from the `LoginAsync` exactly as-is